### ISSUE TITLE
GEEM value specification for categorical tree - semantics of selecting underlying option

### ISSUE THREAD

**Issue by Public-Health-Bioinformatics** [GenEpiO/geem#18]
For comment:

If one only selects an underlying option of a categorical specification - e.g. "bacteria" under "specimen organism taxonomy (species)", should that be interpreted minimally as a categorical tree variable for inclusion as a stand alone item?  Yes, if no parent items are selected. If however "specimen organism taxonomy (species)" was also selected (as shown below), then this suggests that "bacteria" is redundantly selected under that broader specification.  "bacteria" can then be set to be "excluded", or have further annotation on it by user other than "inclusion/exclusion" status. 

<img alt="screen shot 2018-07-22 at 1 57 39 pm" src="https://user-images.githubusercontent.com/10779910/43050058-930bfde8-8db7-11e8-8dbb-3d70c6893708.png">

Here is the shopping cart view:

<img alt="screen shot 2018-07-22" src="https://user-images.githubusercontent.com/10779910/43050197-b9f4fb42-8db9-11e8-8a0f-0f259ee1deb0.png">

So in conclusion categorical items can be set to be included even when their ancestor spec. status is also "include".
